nautobot/utilities/views.py

```diff
@@ -100,6 +100,10 @@ def has_permission(self):
 
         return False
 
+    # TODO(jathan): We may not need this anymore. But we do want to consider the
```

Comment:
Let's quantify "may not need this anymore". It would be good to know the criteria for when we can safely remove this.

Reply:
Now that we've moved away from the legacy generic views, ALL of this code can just go away. This comment can just be removed now since it will only be around for the remainder of 1.x